### PR TITLE
[Admin] implemented buttons to approve or reject a new committee.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: src/AppBundle/Admin/CommitteeAdmin.php [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+                    'disabled' => true,
+                ])
+                ->add('postAddress.longitude', TextType::class, [
+                    'label' => 'Longitude',
+                    'disabled' => true,
+                ])
+            ->end()
+        ;
+
+        /*$showMapper
```
Comment (tgalopin) [inline src/AppBundle/Admin/CommitteeAdmin.php]: Forgotten commented code?

File: src/AppBundle/Controller/Admin/AdminCommitteeController.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+use AppBundle\Entity\Committee;
+use AppBundle\Exception\CommitteeException;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("/committee")
```
Comment (tgalopin) [inline src/AppBundle/Controller/Admin/AdminCommitteeController.php]: Can you put it under `/admin` to ensure security control?

Comment (hhamon) [inline src/AppBundle/Controller/Admin/AdminCommitteeController.php]: It's already the case, look at the import in `routing.yml` file ;)

Comment (tgalopin) [inline src/AppBundle/Controller/Admin/AdminCommitteeController.php]: Then please remove it from https://github.com/EnMarche/en-marche.fr/blob/feat-admin-approve-committee/src/AppBundle/Controller/Admin/AdminAdherentController.php

**Pull request merged**